packages/device-client/src/lib/types/index.ts

```diff
@@ -4,6 +4,11 @@
  * This software may be modified and distributed under the terms
  * of the MIT license. See the LICENSE file for details.
  */
+
+// Re-export types from external dependencies that consumers need
+export type { ConfigOptions } from '@forgerock/javascript-sdk';
```

Comment:
technically this package uses the javascript-sdk. we can update this later though.